react/src/reactTextMask.js

```diff
@@ -52,12 +37,12 @@ export const MaskedInput = React.createClass({
     return (
       <input
         {...props}
-        onInput={this.onChange}
+        onInput={event => this.onChange(event)}
```

Comment:
It shouldn't be needed. As I understand it, the old version may perform a little better.